gnocchi/rest/aggregates/api.py

```diff
@@ -567,6 +701,11 @@ def groupper(r):
 
         results = []
         for key, resources in itertools.groupby(resources, groupper):
+            resources = list(resources)
+
+            LOG.debug("Executing collection of metrics for group [%s] and "
+                      "resources [%s].", key, resources)
```

Comment:
```suggestion
            LOG.debug("Executing collection of metrics for group [%s] and "
                      "resources [%s].", key, list(resources))
```

Reply:
Why would this change be better than the way it is coded now? If resources is an iterator, when we create the list here, it will be empty, and then when the resources variable is used, there will be nothing there.

Reply:
valid point :)